doc/interop-test-descriptions.md

```diff
@@ -418,21 +420,32 @@ Procedure:
       }
     }
     ```
+The client attaches custom metadata with the following keys:
+    ```
+    "x-grpc-test-echo-initial"
+    or
```

Comment:
I don't understand this "or". What is the decision based on? If the client only attaches one, then how are both asserted?